blackmagic/db.py

```diff
@@ -72,11 +78,13 @@ def dump_sync(self):
         image.flavour = 'C'
         image.status = self._status
 
-        configuration_props = self._get_configuration_props()
-        image.props = {
+        props = {
             'PIEMAN_INCLUDES': ','.join(self._selected_packages),
-            **configuration_props,
         }
+        props.update(self._get_configuration_props())
```

Comment:
I guess the changes are redundant. You simply needed to replace `image.props` for `props` and that's it.